apex-checks/src/main/java/org/fundacionjala/enforce/sonarqube/apex/checks/TestAssertionsAndTestMethodKeywordCheck.java

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Yes, the 'name' property should be put in the 'apex.properties' file using the key attribute and the error message should be put in the 'ChecksResourceBundle.properties' file